contracts/tokenbridge/ethereum/L1AtomicTokenBridgeCreator.sol

```diff
@@ -45,6 +46,12 @@ contract L1AtomicTokenBridgeCreator is Initializable, OwnableUpgradeable {
     // non-canonical router registry
     mapping(address => address) public inboxToNonCanonicalRouter;
 
+    // hard-code it to make sure gas limit is big enough for deployment to succeed
```

Comment:
maybe also explain why we need it hardcoded

Reply:
done